test/views/html/incomes/nextYear/updateIncomeCYPlus1StartSpec.scala

```diff
@@ -39,7 +40,16 @@ class updateIncomeCYPlus1StartSpec extends TaiViewSpec {
       doc(view).getElementsByClass("button").text must include(messages("tai.income.details.updateTaxableIncome.update"))
     }
 
+    "contain the correct content when income is from pension" in {
```

Comment:
These is a possible improvement to the tests (may be over the top).

So the main question is what happens if you accidentally removed the if statement from the view? 

I believe that all of these tests would still pass. Perhaps we should be testing the negative too:
`doc(view).getElementsByTag("p").text mustNot include(messages("tai.updateIncome.CYPlus1.start.paragraph2", employerName))`

Reply:
Done